R/position-dodge.R

```diff
@@ -104,7 +106,10 @@ PositionDodge <- ggproto("PositionDodge", Position,
   preserve = "total",
   orientation = "x",
   reverse = NULL,
+  default_aes = aes(order = NULL),
```

Comment:
I can be a bit worried that `order` is so general that it conflicts with aesthetics from extension packages. Have you looked into that?

Reply:
It is hard to search for, but I can confidently tell you that no package on CRAN has a `scale_order_*()` function. I've personally never seen an extension package use an `order` aesthetic.